govc/cli/command.go

```diff
@@ -43,16 +44,25 @@ type Command interface {
 	Run(ctx context.Context, f *flag.FlagSet) error
 }
 
-func generalHelp(w io.Writer) {
-	fmt.Fprintf(w, "Usage of %s:\n", os.Args[0])
+func generalHelp(w io.Writer, filter string) {
+	if filter == "" {
+		fmt.Fprintf(w, "Usage of %s:\n", os.Args[0])
+	} else {
+		fmt.Fprintf(w, "Usage of %s filtered by %s\n", os.Args[0], filter)
```

Comment:
:+1: this is useful.  Can we change the wording a bit though, maybe borrowing from ubuntu(?):

```% nope
No command 'nope' found, did you mean:
... list of matching commands ...
```

``` govc tags
govc: command 'tags' not found, did you mean:
  disk.tags.attach
  disk.tags.detach
  tags.attach
  tags.attached.ls
  tags.category.create
  tags.category.info
  tags.category.ls
  tags.category.rm
  tags.category.update
  tags.create
  tags.detach
  tags.info
  tags.ls
  tags.rm
  tags.update
```

Reply:
yup easy peasy